src/Compilers/Core/Portable/Compilation/SemanticModel.cs

```diff
@@ -64,6 +64,39 @@ public SyntaxTree SyntaxTree
         protected abstract SyntaxTree SyntaxTreeCore { get; }
 
         /// <summary>
+        /// Gets the low-level operation corresponding to the method's body.
+        /// </summary>
+        /// <param name="method">The method symbol.</param>
+        /// <param name="cancellationToken">An optional cancellation token.</param>
+        /// <returns></returns>
+        public IOperation GetOperation(IMethodSymbol method, CancellationToken cancellationToken = default)
```

Comment:
>public IOperation GetOperation(IMethodSymbol method, CancellationToken cancellationToken = default) [](start = 8, length = 99)

It is not clear why this API is on SemanticModel. SemanticModel is created for a particular syntax tree, this API doesn't depend on syntax at all. Consider moving it to Compilation.